src/sentry/static/sentry/app/views/organizationEvents/index.jsx

```diff
@@ -71,7 +68,25 @@ class OrganizationEventsContainer extends React.Component {
       updateParams: this.updateParams,
     };
 
-    this.state = OrganizationEventsContainer.getInitialStateFromRouter(props);
+    const values = OrganizationEventsContainer.getStateFromRouter(props);
+    this.state = {
+      ...values,
+      queryValues: {
+        ...values,
```

Comment:
I'm not sure I follow why all the key-value pairs in `values` needs to be inserted into state twice (here and line 73). Could we skip the ...values above and keep them all under queryValues?

Reply:
we keep projects/environments in state for the multiple selector components and they don't get written into queryValues until the selector is blurred.